modules/cluster-logging-deploy-cli.adoc

```diff
@@ -54,24 +51,16 @@ endif::[]
 metric, which would cause conflicts.
 <2> String. You must specify this label as shown to ensure that cluster monitoring scrapes the `openshift-operators-redhat` namespace.
 
-.. Create the namespace:
-+
-[source,terminal]
-----
-$ oc create -f <file-name>.yaml
-----
-+
-For example:
+. Apply the `Namespace` object by running the following command:
 +
 [source,terminal]
 ----
-$ oc create -f eo-namespace.yaml
+$ oc apply -f <filename>
```

Comment:
Wouldn't hurt to footnote this user value with what to substitute with here. I would likely do this for each command in each section. Perhaps you can make a generic note? "Insert the filename of the object you just created." or something like that?

Reply:
This seems to be pretty hit or miss throughout the docs wrt. whether there's a standard here. There are some instances where people seem to add footnotes, but plenty where they're not added. I would err on the side of I think we can assume that level of knowledge, and this is what has been done for any other doc set I've worked on previously, so I'd prefer not to start adding footnotes for this unless we end up adding a rule in the doc guidelines or somewhere about it.